src/utils/constants.ts

```diff
@@ -4,6 +4,7 @@ export const CALYPTIA_API_VALIDATION_PATH = 'v1/config_validate/fluentbit';
 
 export const PROBLEM_MATCHER_FILE_NAME = 'problem-matcher.json';
 
+export const FALSE_VALUE = 'false';
```

Comment:
This is annoying, we really have to compare against a false string?

Reply:
yes. we do. it is annoying.